src/app/calculator/steam/tank-insulation-reduction/tank-insulation-reduction.service.ts

```diff
@@ -76,10 +78,12 @@ export class TankInsulationReductionService {
       } else {
         form.controls.customInsulationConductivity.clearValidators();
       }
+      form = this.setSurfaceTempValidators(form);
```

Comment:
Looks like this ln 81 will override the validation set if material selection == 1 on ln 77.